src/promptflow/promptflow/azure/_pf_client.py

```diff
@@ -153,7 +153,17 @@ def run(
 
         .. note:: at least one of data or run must be provided.
 
-        .. admonition:: column_mapping
+        .. admonition::
+            Data can be local file or remote path.
+            - Example:
+                - `data = "path/to/local/file"`
```

Comment:
shall we add tests for all these type of data?

Reply:
yes, currently there's existing test for them: e.g. `test_submit_run_from_yaml` in `test_flow_run.py`